### PR TITLE
WIP: Our Team Page

### PR DESCRIPTION
**Pull request by Eli-CB**
## Overview

Details about us and the acknowledgement. Done in order to fit the website rubric.

## Screenshots

ourteam1
ourteam2

## Review

@AdCLeung @arshiannafi @sdevalapurkar

### CODE REVIEW

File: pages/ourteam.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/*------------- HEADER -------------------*/
+
```
Comment (sdevalapurkar) [inline pages/ourteam.css]: remove space

File: pages/ourteam.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.us p.us-title {
+    font-size: 2.4rem;
+    padding-bottom: 5px;
+    margin: 0;
+}
+
+/*------------- ACKNOWLEDGEMENT -------------------*/
+
```
Comment (sdevalapurkar) [inline pages/ourteam.css]: remove space

File: pages/ourteam.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+import Container from 'react-bootstrap/Container';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Button from 'react-bootstrap/Button';
+import Background from '../img/background.jpeg';
+import axios from 'axios';
+import './ourteam.css';
+
+class Ourteam extends Component {
+
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: remove space

File: pages/ourteam.js [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+      if (authToken) {
+          axios.post(`${host}:${port}/authstatus`, { headers: { 'Authorization' : `Bearer ${authToken}` } }).then(res => {
+              if (res.status === 200) {
+                  this.setState({ isAuthenticated: true, name: res.data.authData.name });
+              }
+          }).catch(err => {});
+      }
+  }
+
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: remove one space here

File: pages/ourteam.js [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+  render() {
+    return (
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: tabbing in this section (4 spaces)

File: pages/ourteam.js [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: remove space

File: pages/ourteam.js [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+                  We would like to thank Mihai Sima for
+                   his help with the grading and design.
+                  Professor Mihai Sima specializes in embedded systems and electrical engineering.
+                  </Card.Text>
+                  <Card.Link href="https://www.ece.uvic.ca/~msima/">Website</Card.Link>
+                </Card.Body>
+              </Card>
+
+            </div>
+
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: extra spaces here, could prob remove

File: pages/ourteam.js [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+          </main>
+          <div className="m-5"/>
+      </div>
+
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: same here

File: pages/ourteam.js [hunk 9]
```diff
@@ -64,8 +64,8 @@ class Ourteam extends Component {
               <h1>Our Team</h1>
               <div>
                 <p>
-                  We are a team of five University of Victoria Engineering students.
-                  Brakes Supreme was created as part of our capstone project course: ECE/SENG 499.
+                  We are a team of six University of Victoria Engineering students who formed
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: good fix here, just noticed it now in the screenshot and was about to comment on it but then I saw your fix 👍

File: pages/ourteam.js [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+              integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp"
+              crossorigin="anonymous"
+          />
+          <link
+              href="https://fonts.googleapis.com/css?family=Raleway"
+              rel="stylesheet"
+          />
+          <header>
+            <NavbarBootstrap
+                isAuthenticated={false}
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: this prop should not be static `false`, it should be based on the value of `this.state.isAuthenticated`

Comment (sdevalapurkar) [reply pages/ourteam.js]: you also need to pass in the name prop

File: pages/ourteam.js [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+                  </p>
+                </div>
+                <div className="us-three">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/adam.jpg')}  />
+                    <p className="us-title">Adam</p>
+                    <p>Software/Hardware</p>
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: Maybe we can give more descriptive titles to everyone, like `lead software architect` or `embedded systems engineer`
what do you think @Eli-CB ?

File: pages/ourteam.js [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+                  width={340}
+                  height={240}
+                />
+              </h1>
+              <Card style={{ width: '40rem' }} bg="#f8f9fa" border="dark">
+                <Card.Body>
+                  <Card.Title>Mihai Sima</Card.Title>
+                  <Card.Subtitle className="mb-2 text-muted">Supervisor</Card.Subtitle>
+                  <Card.Text>
+                  We would like to thank Mihai Sima for
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: i think it should be `Dr. Mihai Sima`

File: pages/ourteam.js [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+                   his help with the grading and design.
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: `grading and design of our project. Our project would not have been successful had it not been for his input and support.`

File: pages/ourteam.js [hunk 14]
```diff
@@ -0,0 +1,180 @@
...
+                  Professor Mihai Sima specializes in embedded systems and electrical engineering.
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: `Dr. Mihai Sima specializes in Embedded Systems and Electrical Engineering, and is a professor at the University of Victoria`

File: pages/ourteam.js (hunk 11 above)
Comment (Eli-CB) [reply pages/ourteam.js]: Agreed. The titles were temporary. I'll think of something

File: pages/ourteam.js [hunk 15]
```diff
@@ -1,180 +1,154 @@
 import React, { Component } from 'react';
 import NavbarBootstrap from '../components/NavbarBootstrap';
 import Card from 'react-bootstrap/Card';
-import Jumbotron from 'react-bootstrap/Jumbotron';
 import Image from 'react-bootstrap/Image';
-import Figure from 'react-bootstrap/Figure';
-import Container from 'react-bootstrap/Container';
```
Comment (sdevalapurkar) [inline pages/ourteam.js]: nice! dont need imports if we dont use em

**Pull request merged**